web/src/routes/TagDetails/TagDetailsTabs.tsx

```diff
@@ -59,6 +64,13 @@ export default function TagTabs(props: TagTabsProps) {
       >
         <Packages />
       </Tab>
+      <Tab
+        eventKey={TabIndex.ModelCard}
+        title={<TabTitleText>ModelCard</TabTitleText>}
+        isHidden={!quayConfig.features.UI_MODELCARD || !props.tag.modelcard}
```

Comment:
this may not load right away, it will need to be `!quayConfig?.features?.UI_MODELCARD`